src/backend/TrafficCourts/Citizen.Service/Features/Disputes/Create.cs

```diff
@@ -91,21 +112,22 @@ public async Task<Response> Handle(Request request, CancellationToken cancellati
                     submitNoticeOfDispute.ViolationTicket = _mapper.Map<Messaging.MessageContracts.ViolationTicket>(lookedUpViolationTicket);
                 }
                 submitNoticeOfDispute.CitizenSubmittedDate = DateTime.UtcNow;
-                
-                var response = await _submitDisputeRequestClient.GetResponse<DisputeSubmitted>(submitNoticeOfDispute);
 
                 // Send email message to the submitter's entered email
                 var template = MailTemplateCollection.DefaultMailTemplateCollection.FirstOrDefault(t => t.TemplateName == "SubmitDisputeTemplate");
                 if (template is not null)
                 {
-                    _sendEmailRequestClient.Create(new SendEmail()
+                    var emailMessage = new SendEmail()
                     {
                         From = template.Sender,
                         To = { createDisputeRequest.EmailAddress },
-                        Subject = template.SubjectTemplate,
+                        Subject = template.SubjectTemplate.Replace("<ticketid>", createDisputeRequest.TicketNumber),
                         PlainTextContent = template.PlainContentTemplate?.Replace("<ticketid>", createDisputeRequest.TicketNumber)
-                    }, cancellationToken);
+                    };
+                    _sendEmailRequestClient.Create(emailMessage, cancellationToken);
                 }
+                
+                var response = await _submitDisputeRequestClient.GetResponse<DisputeSubmitted>(submitNoticeOfDispute);
```

Comment:
I think there is a JIRA ticket to move this email logic out of this method and citizen-api project completely (including the MailTemplateCollection.cs model class) and into an email consumer worker in the Workflow service project instead.